### PR TITLE
Update documentation

### LINKED ISSUE #15: Update User Guide images

### LINKED ISSUE #124: Update documentation with information from v1.2

### PR DESCRIPTION
**Pull request by samuelfangjw**
closes #124, closes #15 
view updated user guide at https://www.samuelfangjw.com/tp/

### CODE REVIEW

**Comment by samuelfangjw** [comment]
Ready for review

File: docs/UserGuide.md [hunk 1]
```diff
@@ -21,16 +21,18 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`list`** and pressing Enter will list all contacts.<br>
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`listC`** and pressing Enter will list all contacts.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`viewP 1`** : Displays the first project.
```
Comment (lirc572) [inline docs/UserGuide.md:27]: To be consistent, `1` should not be bold

File: docs/UserGuide.md [hunk 2]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
 
 </div>
 
-### Viewing help : `help`
+#### Viewing help : `help`
```
Comment (lirc572) [inline docs/UserGuide.md]: Perhaps it is better to put this under a level 3 heading named `others` or something? It looks a bit strange to me to have a level 4 heading parallel with level 3 headings.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
 Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
+### Projects
 
-### Adding a person: `add`
+#### Viewing a project: `listP`
```
Comment (lirc572) [inline docs/UserGuide.md:71]: According to seedu md standard, we should put an empty line after each heading: 
image

File: docs/UserGuide.md [hunk 4]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* Displays a panel with details of the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `viewP 1` Displays the first project. 
+* `viewP 2` Displays the second project.
+
+#### Viewing the events tab: `tabE`
+Switches to the events tab of the project that is currently displayed.
```
Comment (lirc572) [inline docs/UserGuide.md:87]: same here

File: docs/UserGuide.md [hunk 5]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+
+Format: `tabE`
+
+* This command can only be used when a project is currently being displayed.
+
+#### Viewing the deadlines tab: `tabD`
+Switches to the deadlines tab of the project that is currently displayed.
```
Comment (lirc572) [inline docs/UserGuide.md:95]: same here

File: docs/UserGuide.md [hunk 6]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+
+Format: `tabD`
+
+* This command can only be used when a project is currently being displayed.
+
+#### Viewing the todos tab: `tabT`
+Switches to the todos tab of the project that is currently displayed.
```
Comment (lirc572) [inline docs/UserGuide.md:103]: same here

File: docs/UserGuide.md [hunk 7]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+
+Format: `tabT`
+
+* This command can only be used when a project is currently being displayed.
+
+#### Viewing the participants tab: `tabP`
+Switches to the participants tab of the project that is currently displayed.
```
Comment (lirc572) [inline docs/UserGuide.md:111]: same here

File: docs/UserGuide.md [hunk 8]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+
+Format: `tabP`
+
+* This command can only be used when a project is currently being displayed.
+
+#### Adding a project: `addP`
+Adds a project to CoLAB.
```
Comment (lirc572) [inline docs/UserGuide.md:119]: same here

File: docs/UserGuide.md [hunk 9]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+
+Format: `addP n/PROJECT_NAME`
+
+Examples:
+* `addP n/CS2103T Team Project`
+* `addP n/CS2101 OP2`
+
+#### Adding an event to a project: `addEto`
+Adds an event to a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:129]: same here

File: docs/UserGuide.md [hunk 10]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+    * `MONTHLY`
+    * `YEARLY`
+* `REPEATABLE_DATE` must be in `dd-MM-yyyy` format.
+
+Examples:
+* `addEto 1 d/Project Meeting i/WEEKLY at/24-04-2021`
+* `addEto 2 d/CS2101 Presentation i/NONE at/24-04-2021`
+
+#### Adding a deadline to a project: `addDto`
+Adds a deadline to a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:151]: same here

File: docs/UserGuide.md [hunk 11]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu. 
+* The index **must be a positive integer** 1, 2, 3, …​
+* `REPEATABLE_DATE` must be in `dd-MM-yyyy` format.
+
+Examples:
+* `addDto 1 d/Milestone v1.2 by/01-03-2021`
+* `addDto 2 d/CS2101 Project Submission by/15-03-2021`
+
+#### Adding a todo to a project: `addTto`
+Adds a todo to a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:166]: same here

File: docs/UserGuide.md [hunk 12]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* Adds a deadline to the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `addTto 1 d/Add unit tests`
+* `addTto 2 d/Finish slides for presentation`
+
+#### Add a contact to a project: `addCto`
+Adds a contact to a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:180]: same here

File: docs/UserGuide.md [hunk 13]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addCto 1 n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+* `addCto 2 n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+#### Deleting a project: `deleteP`
+Deletes a specified project from CoLAB.
```
Comment (lirc572) [inline docs/UserGuide.md:198]: same here

File: docs/UserGuide.md [hunk 14]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The index **must be a positive integer** 1, 2, 3, …​
+  
+Examples:
+* `deleteP 1`
+* `deleteP 2`
 
-### Editing a person : `edit` `[coming soon]`
+#### Deleting an event from a project: `deleteE`
+Deletes a specified event from a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:212]: same here

File: docs/UserGuide.md [hunk 15]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e. adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Examples:
+* `deleteE 1 r/1`
+* `deleteE 2 r/2`
+
+#### Deleting a deadline from a project: `deleteD`
+Deletes a specified deadline from a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:227]: same here

File: docs/UserGuide.md [hunk 16]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* Both indexes **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `deleteD 1 r/1`
+* `deleteD 2 r/2`
+
+#### Deleting a todo from a project: `deleteT`
+Deletes a specified todo from a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:242]: same here

File: docs/UserGuide.md [hunk 17]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `TODO_INDEX` refers to the number shown beside the todo when viewing the project.
+* Both indexes **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `deleteT 1 r/1`
+* `deleteT 2 r/2`
+
+#### Deleting a contact from a project: `deleteCfrom`
+Deletes a specified contact from a specified project.
```
Comment (lirc572) [inline docs/UserGuide.md:257]: same here

File: docs/UserGuide.md [hunk 18]
```diff
@@ -64,53 +66,223 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
 
-### Locating persons by name: `find`
+Format: `deleteCfrom PROJECT_INDEX r/CONTACT_INDEX`
```
Comment (lirc572) [inline docs/UserGuide.md:259]: same here

**Review by lirc572 — CHANGES_REQUESTED** [review]
Some minor style issues.

**Comment by samuelfangjw** [comment]
> Some minor style issues.

Thanks for pointing this out,  wasn't aware there was a style guide for md files. Will look through the style guide again and update.

File: docs/UserGuide.md [hunk 19]
```diff
@@ -64,53 +66,230 @@ CoLAB (Command Line Address Book) is a **desktop app for students currently enro
...
+* Deletes the contact at the specified `CONTACT_INDEX` from the project at the specified `PROJECT_INDEX`.
+* The `PROJECT_INDEX` refers to the number shown beside the project in the side menu.
+* The `CONTACT_INDEX` refers to the number shown beside the contact when viewing the project.
+* Both indexes **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `deleteCfrom 1 r/1`
+* `deleteCfrom 2 r/2`
+
+### Contacts Lists
```
Comment (lirc572) [inline docs/UserGuide.md]: Should this be `Contact List` instead since we have only one such list?

**Review by lirc572 — CHANGES_REQUESTED** [review]
The changes look good! Just one more minor problem.

File: docs/UserGuide.md (hunk 19 above)
Comment (samuelfangjw) [reply docs/UserGuide.md]: Agreed, thanks for being thorough.

Comment (samuelfangjw) [reply docs/UserGuide.md]: Changed it to contacts to be more consistent with the other level 3 heading  (projects).  Do you think this  is ok?

Comment (lirc572) [reply docs/UserGuide.md]: Yes! That would be nice!

**Review by lirc572 — APPROVED** [review]

**Pull request merged**